client/src/modules/Schedule/components/FilteredTags/FilteredTags.tsx

```diff
@@ -0,0 +1,21 @@
+import { Tag } from 'antd';
+import { FilterFilled } from '@ant-design/icons';
+import { TAG_NAME_MAP } from 'modules/Schedule/constants';
+import { CourseScheduleItemDto } from 'api';
+
+type FilteredTagsProps = {
+  tagFilter: string[];
+  onTagClose: (tag: string) => void;
+};
+
+export const FilteredTags = ({ tagFilter, onTagClose }: FilteredTagsProps) =>
+  tagFilter?.length > 0 ? (
+    <div style={{ marginBottom: 14 }}>
```

Comment:
usually we try to use margins/padding that based on 4 -> 4, 8, 16, 32 etc

Reply:
Thank you. Fixed